pkg/cloud/services/compute/instance.go

```diff
@@ -508,10 +504,22 @@ func (s *Service) getOrCreatePort(eventObject runtime.Object, clusterName string
 	}
 
 	record.Eventf(eventObject, "SuccessfulCreatePort", "Created port %s with id %s", port.Name, port.ID)
+
+	if portOpts.Trunk {
+		trunkDescription := names.GetDescription(clusterName)
+		trunk, err := s.getOrCreateTrunk(eventObject, trunkDescription, port.Name, port.ID)
```

Comment:
Yes, this PR is creating a new trunk for every port marked as trunked. 
However, it is configurable as is shown in the description of this PR.

Reply:
Please ignore this because it's out of scope for your change and consistent with the previous behaviour. However, now that I've seen it I don't understand why trunks are used this way. That could just be because I don't properly understand trunks, though.

Reply:
@mdbooth: It's a Neutron trunk, which might be a bit confusing. In Neutron a *port* can have a trunk created and then you add other ports as *subports* in that trunk. In Kuryr we use that to dynamically add subports which serve as pods interfaces. What you're proposing is doable, but I'm not sure if I see value in subports added statically. Anyway if we'd want to implement that, I'd rather add a `subports` field in `PortOpts` and get subports defined there.

Reply:
@dulek is the name of the trunk singinificant to kuryr? Does kuryr look up trunks per port (parent port.name for example), or by instance